src/logics/radiant-v2/tokens/index.ts

```diff
@@ -1,11 +1,9 @@
 import arbitrumTokensJSON from './data/arbitrum.json';
 import * as common from '@protocolink/common';
-import mainnetTokensJSON from './data/mainnet.json';
 
 type ArbitrumTokenSymbols = keyof typeof arbitrumTokensJSON;
 
-export const arbitrumTokens = common.toTokenMap<ArbitrumTokenSymbols>(arbitrumTokensJSON);
-
-type MainnetTokenSymbols = keyof typeof mainnetTokensJSON;
-
-export const mainnetTokens = common.toTokenMap<MainnetTokenSymbols>(mainnetTokensJSON);
```

Comment:
Such as `protocolink-api/test/controllers/radiant-v2/withdraw.test.ts` uses mainnetTokens for testing, so that we should still export those mainnet rTokens.

But actually we can remove unused `wstETH` in `protocolink-logics/src/logics/radiant-v2/tokens/data/mainnet.json`.

Reply:
added `mainnetTokensJSON` back, add removed `wstETH`